packages/now-cli/test/integration.js

```diff
@@ -519,6 +519,26 @@ test('Deploy `api-env` fixture and test `now env` command', async t => {
   await nowEnvLsIsEmpty();
 });
 
+test('deploy with metadata with = as the value', async t => {
+  const target = fixture('redirects-v2');
+
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [target, ...defaultArgs, '--confirm', '--meta', 'someKey=='],
+    {
+      reject: false,
+    }
+  );
+
+  t.is(exitCode, 0, formatOutput({ stderr, stdout }));
+
+  const i = stdout.lastIndexOf('https://');
+  const url = stdout.slice(i);
```

Comment:
Is this slicing required? The _only_ thing that `now deploy` should be printing to stdout is the URL. If there's extra stuff being printed then that's a bug.

Reply:
Fixed in https://github.com/zeit/now/pull/4160/commits/d2156dfb98574861564c7fd24a631bd11dd4c063.